detectable/src/main/java/com/blackduck/integration/detectable/detectables/gradle/inspection/parse/GradleReportTransformer.java

```diff
@@ -24,22 +28,108 @@
+        for (GradleTreeNode currentNode : configuration.getChildren()) {
+            if (treeNodeSkipper.shouldSkip(currentNode)) {
+                continue;
+            }
+
+            if (currentNode.getNodeType() == GradleTreeNode.NodeType.GAV) {
+                addGavToGraph(currentNode, currentNode.getLevel(), history, rootGraph);
+            }
+            else if (currentNode.getNodeType() == GradleTreeNode.NodeType.PROJECT) {
+                // Let this method skip over the subProject section as usual
+                treeNodeSkipper.skipUntilLineLevel(currentNode.getLevel());
+                // Process subProject section separately
+                processSubprojectAndCreateCodeLocation(currentNode, configuration.getChildren(), rootReport);
             } else {
-                logger.trace("Excluding unresolved configuration from the graph: {}", configuration.getName());
+                treeNodeSkipper.skipUntilLineLevel(currentNode.getLevel());
             }
         }
+    }
+
+    private void processSubprojectAndCreateCodeLocation(GradleTreeNode subProjectNode, List<GradleTreeNode> allTreeNodesInCurrentConfiguration, GradleReport rootReport) {
+        String subProjectName = subProjectNode.getProjectName().get();
+        logger.debug("Processing subProject node:" + subProjectName);
+        int subProjectSectionStartIndex = allTreeNodesInCurrentConfiguration.indexOf(subProjectNode);
+        int subProjectNodeLevel = subProjectNode.getLevel();
+
+        DependencyGraph subProjectGraph = new BasicDependencyGraph();
+        DependencyHistory history = new DependencyHistory();
+        TreeNodeSkipper treeNodeSkipper = new TreeNodeSkipper();
+        // Begin processing subProject section
+        for (int i = subProjectSectionStartIndex+1; i < allTreeNodesInCurrentConfiguration.size(); i++) {
+            GradleTreeNode currentNode = allTreeNodesInCurrentConfiguration.get(i);
+            int currentNodeLevelRelativeToSubProject = (currentNode.getLevel() - 1) - subProjectNodeLevel;
+            if (treeNodeSkipper.shouldSkip(currentNode)) {
+                continue;
+            }
+            if (currentNodeLevelRelativeToSubProject != -1) {
+                if (currentNode.getNodeType() == GradleTreeNode.NodeType.GAV) {
+                    logger.trace("Adding dependency " + currentNode.getGav() + " for subProject " + subProjectName);
+                    addGavToGraph(currentNode, currentNodeLevelRelativeToSubProject, history, subProjectGraph);
+                } else {
+                    // Process nested subproject node
+                    treeNodeSkipper.skipUntilLineLevel(currentNode.getLevel());
+                    processSubprojectAndCreateCodeLocation(currentNode, allTreeNodesInCurrentConfiguration, rootReport);
+                }
+            } else {
+                // Current node is back at 0 so we are done processing subProject section
```

Comment:
Debug log?

Reply:
Added trace logs around subproject section processing. Will be useful if needed to troubleshoot in prod, thanks!